submissions/ik-web/oop-exercise/index.js

```diff
@@ -0,0 +1,63 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/ik-web/a-tiny-JS-world
+   Web app: https://ik-web.github.io/a-tiny-JS-world/
+
+*/
+
+// ======== OBJECTS DEFINITIONS ========
+class Inhabitant {
+    constructor(species, name, gender, legs, hands, saying) {
+       this.species = species;
+       this.name = name;
+       this.gender = gender;
+       this.legs = legs;
+       this.hands = hands;
```

Comment:
Not all inhabitants have hands by natural design. The beauty of OOP is that every base class defines and handles only the props that are shared by its descendants, and every child class introduces and handles props that are unique to it and its descendants.

Reply:
I fixed this moment